### PR TITLE
Suppress logging

### PR DESCRIPTION
**Pull request by daniel-thom**
The main purpose of this PR is to suppress `Info` log messages from showing up in the notebook. I think I heard from you guys that you would prefer this behavior. I made the changes in a subset of notebooks. If you agree with the changes then I'll continue with the rest.

I also replaced JSON2 with JSON3.

### CODE REVIEW

File: script/2_PowerSystems_examples/add_forecasts.jl [hunk 1]
```diff
@@ -22,14 +26,9 @@ include(joinpath(pkgpath, "test", "2_PowerSystems_examples", "parse_matpower.jl"
...
-    end
+    @JSON3.@pretty JSON3.read(f)
 end
 
-# ### Read the pointers
-ts_pointers = PowerSystems.IS.read_time_series_file_metadata(fname)
-
-# ### Read and assign time series to `System` using the `ts_pointers` struct
-add_time_series!(sys, ts_pointers)
+# ### Read and assign time series to `System` using these parameters.
```
Comment (daniel-thom) [inline script/2_PowerSystems_examples/add_forecasts.jl:32]: @claytonpbarrows IMHO it is better to keep this simpler. If you do want to expose the intermediate step to users, I'll revert the change.

File: script/2_PowerSystems_examples/serialize_data.jl [hunk 2]
```diff
@@ -10,17 +10,19 @@
...
 # ### Write data to a temporary directory
 
 folder = mktempdir()
 path = joinpath(folder, "system.json")
-@info "Serializing to $path"
+println("Serializing to $path")
 to_json(sys, path)
 
-filesize(path) / 1000000 #MB
+filesize(path) / (1024 * 1024) #MiB
```
Comment (daniel-thom) [inline script/2_PowerSystems_examples/serialize_data.jl:25]: Tools like `ls` usually provide these sizes in base 2, not base 10. 1 Mebibyte = 2^20 bytes

**Review by jd-lara — APPROVED** [review]
Thanks, @daniel-thom this will make the examples much better. I will see if the same works for the docs too.

File: script/2_PowerSystems_examples/add_forecasts.jl (hunk 1 above)
Comment (jd-lara) [reply script/2_PowerSystems_examples/add_forecasts.jl:32]: I think the same, there are too many intermediate steps here for this to be a tutorial

File: script/2_PowerSystems_examples/PowerSystems_intro.jl [hunk 3]
```diff
@@ -42,7 +42,9 @@ Pkg.status()
 using SIIPExamples;
 using PowerSystems;
 using D3TypeTrees;
-IS = PowerSystems.IS
+using Logging
+
```
Comment (claytonpbarrows) [inline script/2_PowerSystems_examples/PowerSystems_intro.jl:46]: can we add something like the following:
```
# PowerSystems provides some control over log messages, [this doc page](https://nrel-siip.github.io/PowerSystems.jl/stable/api/public/#logging) provides more information about how to configure the logger.
```

**Review by claytonpbarrows — APPROVED** [review]
This looks good. Thanks!

**Pull request merged**